### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by Stonedestroyer**
### Type

- [X] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Yearly grammar PR. 

Co-Author: @thisisjvgrace

### CODE REVIEW

**Review by Drapersniper — DISMISSED** [review]
Slime stop sliming <3

File: redbot/core/commands/commands.py [hunk 1]
```diff
@@ -815,7 +815,7 @@ async def invoke(self, ctx: "Context"):
                 return await ctx.send_help()
         elif self.invoke_without_command:
             # So invoke_without_command when a subcommand of this group is invoked
-            # will skip the the invokation of *this* command. However, because of
+            # will skip the the invocation of *this* command. However, because of
```
Comment (owocado) [inline redbot/core/commands/commands.py]: Might as well want to remove the extra "the" here.

Comment (Stonedestroyer) [reply redbot/core/commands/commands.py]: thanks

**Review by Drapersniper — APPROVED** [review]
LGTM again

**Pull request merged**